src/oceans/ui.jsx

```diff
@@ -1040,14 +1040,12 @@ let Predict = class Predict extends React.Component {
       this.state.timeScale !== defaultTimeScale
     ) {
       selectedControl = MediaControl.FastForward;
-    } else {
-      selectedControl = MediaControl.Play;
```

Comment:
when we set `selectedControl = MediaControl.Play`, it was never being used, so i just removed this case